content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -245,7 +243,15 @@ $ export GATEWAY_SERVICE_ACCOUNT=ns/istio-system/sa/bookinfo-gateway-istio
 
 ## Adding your application to the ambient mesh {#addtoambient}
 
-You can enable all pods in a given namespace to be part of an ambient mesh
+Ambient mesh data plane relies on the ztunnel DaemonSet to redirect traffic. Before we label the namespace to be part of an ambient mesh, check the ztunnel pods to make sure they are in a healthy state:
```

Comment:
```suggestion
Ambient mesh data plane relies on the ztunnel DaemonSet to redirect traffic. Check the ztunnel pods to make sure they are in a healthy state:
```

For clarity. When ztunnel becomes ready, the CNI will "refresh" the configuration so it's not necessary to wait to label the namespace.